package.json

```diff
@@ -64,6 +64,7 @@
     "mermaid": "^8.9.2",
     "moment": "^2.29.1",
     "nspell": "^2.1.5",
+    "react-scripts": "^2.1.8",
```

Comment:
This change is not related to #6 or #9. I suggest don't make changes to `package.json`